src/main/java/svc/models/DemoCitation.java

```diff
@@ -0,0 +1,11 @@
+package svc.models;
+
+import java.time.LocalDate;
+
+public class DemoCitation {
+	public String citationNumber;
+	public LocalDate dob;
+	public String driversLicenseNumber;
+	public Integer numberOfViolations;
+	public Integer daysTillCourt;
```

Comment:
What is this class used for? Also, daysTillCourt is not on the real Citation so why is it on your demo object?